posts/2024-02-01-january-ecodev.mdx

```diff
@@ -0,0 +1,123 @@
+
+
+## Team Updates
+
+The team has expanded this month, with the new EcoDev Team Lead Marina and a new Business Development Lead, Pedro, joining the team.
+
+You can connect with [Marina](https://twitter.com/Crypto_marina) and [Pedro](https://twitter.com/bigbraincap) on X.
+
+If you are interested in working with Waku, [we are currently hiring a Software Engineer](https://boards.greenhouse.io/waku/jobs/5607301) to work with the Waku Chat SDK. The position is fully remote, and we hire globally. 
+
+
+---
+
+
+## Waku Community Call and AMA
+
+Waku will host a Community Call on X on the last Friday of each month. The objective of the call is to update the community on our progress and provide a space where questions can be put to the team 
+
+The first Community Call took place last week on the 26th of January. You can [listen back to the full Space](https://twitter.com/Waku_org/status/1750927368644919722), and read the highlights below. 
+
+[Vaclav](https://twitter.com/vpavlin), [Guru](https://twitter.com/hackyguru), and [Aaron](https://twitter.com/vrycmfy) from the Waku team participated in the call. 
+
+
+### Waku network updates 
+
+**Vaclav**: If you don't know what the Waku Network is, it is our attempt to formalise a network of peer-to-peer nodes that would form some network running Waku protocols. We are very proud to have released the Gen 0, and there will likely be more generations to come right before Devconnect in Istanbul this November.
+
+It's just the beginning for the Waku Network; we're still dogfooding it. You can see [the list of milestones and the roadmap for improvements](https://github.com/waku-org/pm/issues/101) put forward by the research team. There’s also an interesting discussion you can join in the [Vac Forum](https://forum.vac.dev/t/message-validation-marketplace-waku-network-of-networks/262/2).
+
+As part of Generation 0 of Waku Network, we have integrated sharding, ensuring that it’s scalable to thousands of users and that the bandwidth is reasonable for people running on Raspberry Pis or smaller devices with weaker connectivity. Our client targeted at low-resource devices, JSWaku, has been making progress towards fully supporting sharding and, by that, supporting the full Waku Network.
+
+
+### Future incentivisation for running Waku nodes?
+
+**Vaclav: **We are often asked about how we incentivise node runners, as well as who is running the nodes and why. There are currently no monetary or token incentives. However, the research team has recently published findings that offer a glimpse into an incentivisation proposal. That's very interesting and promising for the future of Waku node operators to maybe get something in return.
```

Comment:
I'd say instead that our researchers have defined the problem: https://github.com/waku-org/research/blob/master/docs/incentivization.md and are currently building a first PoC